EarTrumpet/ViewModels/TrayIconViewModel.cs

```diff
@@ -197,6 +197,19 @@ public void SetDefaultAudioDevice(string id)
             _deviceService.SetDefaultAudioDevice(id);
         }
 
+        public void ToggleDefaultAudioDeviceMute()
+        {
+            var defaultDevice = GetAudioDevices().FirstOrDefault(x => x.IsDefault);
```

Comment:
LINQ's `.FirstOrDefault()` will return `null` in the default case (i.e. if there aren't any playback devices available) which needs to be handled otherwise this will result in an exception being thrown when attempting to access things on `defaultDevice`. A simple early-return should be sufficient although I don't know whether that fits with the codebase's code style or not, so one of the maintainers will have to verify that or suggest an alternative.

Reply:
Elsewhere in this file, this case is handled by comparing to a `default` instance, rather than `null`: https://github.com/factormystic/EarTrumpet/blob/2237cc485bad8a160d58cbdc7f7f803120c4e958/EarTrumpet/ViewModels/TrayIconViewModel.cs#L78

Reply:
Hmm, in that case that means either the case where there aren't any playback devices isn't handled anywhere in the codebase and everything breaks horribly or these methods aren't ever called if it's already been determined that there are no devices...

Reply:
GetAudioDevices returns a struct, when calling FirstOrDefault in that cause you get a default "empty" struct if there is no default device.  The code probably won't fail, but it will make a garbage call to Windows with an empty device id further down this function. You could use a default(EarTrumpetAudioDeviceModel) check to see if you have a device to save making that call.

Reply:
Feedback addressed